### ISSUE TITLE
Support adding accounts to lists

### ISSUE THREAD

**Issue by d-jemme** [Lartsch/FediFollow-Chrome#9]
As requested, leaving this here as a potential feature. I've just "debirdified" and i'm adding people I followed on the birdsite to my following list on Mastodon. Unfortunately I can't add people on other instances to lists, but that would be great if I could!

**Comment by Lartsch** [comment]
Thanks for creating the issue. I will look into how this can be achieved. Only issue is that there is no "Add to list" button on public views of external instances, so that button would need to be injected.

**Comment by Freeplayg** [comment]
> Only issue is that there is no "Add to list" button on public views of external instances, so that button would need to be injected.

Couldn't it just be added to this menu?

![Image](https://user-images.githubusercontent.com/40742947/209419215-fd57e06b-974a-4070-9652-9f0f20f0ccf0.png)

**Comment by Lartsch** [comment]
Yes, definitely. That comment was before I simply enabled the 3 dot menu where applicable. On v3 instances, some features are missing because I was too lazy to insert a 3 dot button there for now.
